AdGuard/AdGuard Login Helper/main.m

```diff
@@ -33,7 +33,7 @@ int main(int argc, const char * argv[]) {
                 NSError *error = nil;
                 [NSWorkspace.sharedWorkspace openURLs:@[urlToOpen]
                                  withApplicationAtURL:url
-                                              options:0
+                                              options:NSWorkspaceLaunchWithoutAddingToRecents
```

Comment:
it is actually deprecated:
https://developer.apple.com/documentation/appkit/nsworkspacelaunchoptions/nsworkspacelaunchwithoutaddingtorecents/

Reply:
Thanks for hinting! It's actually the whole method used here which is [deprecated](https://developer.apple.com/documentation/appkit/nsworkspace/1528548-openurls). But I found a [successor](https://developer.apple.com/documentation/appkit/nsworkspace/3172702-openurls) which also provides such an option using the configuration parameter. I'll adjust the PR!